documentation/modules/proc-configuring-kafka-user.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-user-operator.adoc
+
+[id='proc-configuring-kafka-user-{context}']
+= Configuring a Kafka user
+
+Use the properties of the `KafkaUser` resource to configure a Kafka user.
+
+You can use `kubectl apply` to create or modify users, and `kubectl delete` to delete existing users.
+
+* `kubectl apply -f _<user-config-file>_`
+* `kubectl delete KafkaUser _<user-name>_`
+
+When you create a user, you must configure the authentication and authorization mechanisms required when connecting to Kafka brokers using a listener.
+
+This procedure shows how a user is created with TLS authentication.
+The authentication required depends on the xref:assembly-kafka-broker-listener-authentication-deployment-configuration-kafka[type of authentication configured for the Kafka broker listener].
+
+.Prerequisites
+
+* A running Kafka cluster xref:con-mutual-tls-authentication-deployment-configuration-kafka[configured with a Kafka broker listener using TLS authentication and encryption].
```

Comment:
I wonder if everyone will understand that this is not the only option.

```suggestion
* A running Kafka cluster xref:con-mutual-tls-authentication-deployment-configuration-kafka[configured with a Kafka broker listener using for example TLS authentication and encryption].
```

Reply:
I've changed this a little to make it more clear on the types of authentication:

![image](https://user-images.githubusercontent.com/47596553/81048953-31908100-8eb5-11ea-9946-aaafbcdb2a3c.png)

Reply:
That looks good.